app/views/categories/index.html.erb

```diff
@@ -1,5 +1,14 @@
-<% @categories.each do |category| %>
-  <%= render 'partials/category', category: category %>
-<% end %>
+<div class='total'>
+  <h2>Total</h2>
+  <h2>$<%= @total %></h2>
+</div>
 
-<%= link_to 'Add new category', new_category_path %>
\ No newline at end of file
+<div class="cards-contr">
+  <% @categories.each do |category| %>
+    <%= render 'partials/category', category: category %>
+  <% end %>
+</div>
+
+<footer>
+  <%= link_to 'Add new category', new_category_path, class: 'add-button' %>
+</footer>
```

Comment:
- [ ] As stated in the project requirement Kindly make sure that For Home page (categories page) **For each category, the user can see their **name**, **icon** and the **total amount** of all the transactions that belongs to that category.**
- Your categories page Look Likes the below(there is no icon)
![image](https://github.com/Movgto/ror-budget-app/assets/93598189/0cfe2dbb-5aa5-4007-96bf-8a19b2644b22)